langgraph/package.json

```diff
@@ -37,8 +37,9 @@
   "author": "LangChain",
   "license": "MIT",
   "dependencies": {
-    "@langchain/core": ">0.1.61 <0.3.0",
-    "uuid": "^10.0.0"
+    "@langchain/core": ">=0.2.14 <0.3.0",
+    "uuid": "^10.0.0",
+    "zod": "^3.23.8"
```

Comment:
Why do we need an extra dependency now?

Reply:
Very lightly used here to supply a schema - but it's a dependency of `@langchain/core` anyway so figured it wasn't a big deal